net/dns/manager_windows.go

```diff
@@ -118,14 +118,23 @@ func (m windowsManager) setSplitDNS(resolvers []netip.Addr, domains []dnsname.FQ
 // domains can be set without resolvers, which just contributes new
 // paths to the global DNS search list.
 func (m windowsManager) setPrimaryDNS(resolvers []netip.Addr, domains []dnsname.FQDN) error {
-	var ipsv4 []string
-	var ipsv6 []string
-
+	tsGUID, err := windows.GUIDFromString(m.guid)
+	if err != nil {
+		return err
+	}
+	tsLUID, err := winipcfg.LUIDFromGUID(&tsGUID)
+	if err != nil {
+		return err
+	}
+	ips4 := make([]net.IP, 0, len(resolvers))
+	ips6 := make([]net.IP, 0, len(resolvers))
 	for _, ip := range resolvers {
 		if ip.Is4() {
-			ipsv4 = append(ipsv4, ip.String())
+			b := ip.As4()
```

Comment:
If we're upstreaming changes to winipcfg, maybe also propose one to use net/netip types?

Reply:
It already does - https://github.com/tailscale/tailscale/pull/5343